model/transformers/README.md

```diff
@@ -7,7 +7,7 @@ Currently, dffml_model_transformers supports HuggingFace Transformers tensorflow
 
 ## Documentation
 
-Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
+Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers. 
```

Comment:
You're change here added trailing whitespace. We need to remove the trailing whitespace on line 6

```suggestion
Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
```